src/pages/about-us/history.tsx

```diff
@@ -23,14 +35,64 @@ export default function HistoryPage({
   if (!sanityHistoryPage)
     throw `No Sanity document for the culture page was found.`
 
+  // for later:
+  // const theme = useTheme()
+  // const matches = useMediaQuery(theme.breakpoints.up("md"))
+
+  const { timeline } = sanityHistoryPage
+
+  const renderTimeline = () => {
+    return timeline ? (
+      <div>
+        <Stack>
+          {timeline.map((element, index) => {
+            const card = (
+              <Grid item xs={5}>
+                <Typography variant="h6">{element?.title}</Typography>
+                <SanityImage imageAsset={element?.image} />
+                <PortableText content={element?._rawDescription} />
+              </Grid>
+            )
+            const whitespace = <Grid item xs={5} />
```

Comment:
You could in-line this later on rather than defining it outside! Although to be fair maybe this is a nicer way of reading things